package.json

```diff
@@ -78,14 +78,17 @@
   },
   "dependencies": {
     "classnames": "^2.2.0",
-    "cozy-ui": "^3.0.0-beta13",
+    "cozy-bar": "^3.0.0-beta9",
+    "cozy-client-js": "^0.1.3",
+    "cozy-ui": "^3.0.0-beta14",
     "date-fns": "^1.26.0",
     "hammerjs": "^2.0.8",
     "node-polyglot": "^2.2.0",
     "pouchdb": "^6.1.2",
     "pouchdb-find": "^0.10.5",
     "preact": "^7.1.0",
     "preact-compat": "^3.13.1",
+    "react-bosonic": "beta",
```

Comment:
Just a heads up: we shouldn't use non-specific version like this ever again. This more or less caused the bug addressed in #36, and we've been hit by this one other projects before (I've been guilty of doing this too :-)